lib/airtime.js

```diff
@@ -49,22 +49,66 @@ class Airtime {
-                    phoneNumber:Joi.string().required().custom((value)=>{
-                        return phoneValidator(value).isValid? value : new Error(`the phone number ${value} is invalid`);
+     findTransactionStatus(options){
+        return new Promise((resolve, reject) => {
+            const { error, value } = this.validationOptions({ context:'findTransactionStatus', options});
+            if (error) {
+                const combinedMessages = error.details.map(d => d.message).join(';');
+                reject(new Error(combinedMessages));
+                return;
+            } else {
+                const username = this.config.username;
+                const transactionId = value.transactionId;
+                const apikey = this.config.apiKey;
+                
+                axios({
+                    method: 'get',
+                    maxBodyLength: Infinity,
+                    url: `${Common.AIRTIME_URL}/query/transaction/find?username=${username}&transactionId=${transactionId}`,
+                    headers: {
+                        apikey,
+                        Accept: 'application/json',
+                        'Content-Type': 'application/json',
+                    }
+                })
+                    .then(function (resp) {
+                        const httpStatus = resp.status;
+                        if (httpStatus === 200 || httpStatus === 201) {
+                            resolve(resp.data);
+                        } else {
+                            reject(resp.data);
+                        };
+                    })
+                    .catch(function (error) {
+                        return reject(error);
+                    });
+            };
+        })
+    }
+
+    validateOptions({context, options}) {
+        const schema = {
+            send: Joi.object().keys({
+                idempotencyKey: Joi.string().optional(),
+                recipients: Joi.array().items(
+                    Joi.object({
+                        phoneNumber:Joi.string().required().custom((value)=>{
+                            return phoneValidator(value).isValid? value : new Error(`the phone number ${value} is invalid`);
+                        }),
+                        currencyCode: Joi.string().required(),
+                        amount: Joi.number().required(),
                     }),
-                    currencyCode: Joi.string().required(),
-                    amount: Joi.number().required(),
-                }),
-            ).min(1).required(),
-            maxNumRetry: Joi.number().optional(),
-        }).required();
+                ).min(1).required(),
+                maxNumRetry: Joi.number().optional(),
+        }).required(),
+        findTransactionStatus: Joi.object().keys({
+            transactionId: Joi.string().required(),
+        }).required(),
+    }
+
+        if(!schema[context]) throw new Error(`Schema context '${context}' does not exist`);
```

Comment:
@ibrixze This will always throw for [`send()`](https://github.com/AfricasTalkingLtd/africastalking-node.js/pull/120/files#diff-bd2b48c5b178404f342862fb4d43e4c5572f0cd48289456af2b355820a81fb84R15) no?

Reply:
Yes, you're right